oi/type_graph/Types.h

```diff
@@ -175,6 +176,48 @@ struct TemplateParam {
   std::optional<std::string> value;
 };
 
+/*
+ * Incomplete
+ *
+ * A wrapper around a type we couldn't determine the size of.
+ */
+class Incomplete : public Type {
+ public:
+  Incomplete(Type& underlyingType) : underlyingType_(underlyingType) {
+  }
+
+  static inline constexpr bool has_node_id = false;
+
+  DECLARE_ACCEPT
+
+  const std::string& name() const override {
+    return underlyingType_.name();
```

Comment:
I think this should be `void` to generate code that would work as well as it did before.